lib/brains/urls.py

```diff
@@ -2,15 +2,17 @@
 
 from rest_framework.routers import DefaultRouter
 
-from lib.brains.views import buyer
+from lib.brains.views import buyer, paymethod
 
 router = DefaultRouter()
 router.register(r'buyer', buyer.BraintreeBuyerViewSet, base_name='buyer')
```

Comment:
I really liked how you split ``brains.views.*`` into specific modules that matched the URLs closely. Could things like ``mozilla/buyer`` get imported from ``lib.brains.views.mozilla.buyer`` ?